### PR TITLE
Fixing the tabs behavior to navigate the slider

### PR DESCRIPTION
**Pull request by znegrin**
Related to the [Landing Quick Fixes and Improvements](https://docs.google.com/document/d/1rCasMWwPL-VosmWCd98cXdZsQnBVbVjsuYdQvCbjO9c/edit?usp=drive_web&ouid=109385645779064373298), this PR updates the behavior of the slider with the tabs:

image

I added the css and js necessary. 
Thank you @dpordomingo for your help ;)

### CODE REVIEW

**Comment by znegrin** [comment]
Please, @marnovo @ricardobaeta check if everything is fine. I think with this the Engine page is completed and fully functional.

Thanks!

**Comment by dpordomingo** [comment]
here is a screenshot containing the mobile view; imho tabs are not clear enough, and are vertically not aligned.
image

Also, is it a good idea that hover and active tabs has the same style in desktop?
image

File: hugo/layouts/engine/single.html [hunk 1]
```diff
@@ -92,6 +92,22 @@ <h4 class="title-line line-green">Familiar APIs</h4>
                     <h3 class="title-line line-green">Have questions? Ask your codebase</h3>
                     <p>With the source{d} Engine you can start getting answers to tough questions in a matter of minutes with simple SQL queries in your preferred interface. Connecting your data science, business intelligence and software development tools and pipelines takes you even further.</p>
                 </div>
+                <div class="col-sm-12">
+                    <ul class="engine-nav nav pills-nav col-12 justify-content-center flex-column flex-lg-row">
+                        <li id="slider_1" class="nav-item">
```
Comment (dpordomingo) [inline hugo/layouts/engine/single.html]: I'd suggest:
```html
<li data-slide="0" class="nav-item">
    <span class="nav-link active">Software Architecture</span>
</li>
<li data-slide="3" class="nav-item">
    <span class="nav-link">Productivity</span>
</li>
<li data-slide="6" class="nav-item">
    <span class="nav-link">Talent Management</span>
</li>
<li data-slide="8" class="nav-item">
    <span class="nav-link">Security and Compliance</span>
</li>
```

File: src/js/index.js [hunk 2]
```diff
@@ -41,6 +40,27 @@ $(document).ready(function(){
             }
         ]
     });
+
+    $('.engine-nav li span').click(function(){
```
Comment (dpordomingo) [inline src/js/index.js]: I'd suggest:
```js
$(".engine-nav .nav-item").click(function(){
    $('.engine-nav .nav-item span').removeClass('active');
    $(this).children('span').addClass('active');
    $('.engine-cards').slick('slickGoTo', $(this).data('slide'));
});
```
So we would get rid of many lines.

**Comment by znegrin** [comment]
@dpordomingo I just updated the tabs with the suggested code. 

In terms of the other suggestions, I can change to whatever you decide with @marnovo and @ricardobaeta 

Thank you!

**Comment by marnovo** [comment]
[David's comment](https://github.com/src-d/landing/pull/361#issuecomment-462846569) on it is pretty sensible, but as he mentioned worth spinning off as a separate to-do issue (just C&P in a new issue and refer to this one) here not to block the merge.

**Pull request merged**